### PR TITLE
Copy/109 c cc copy sweep

### LINKED ISSUE #109: Sweep for [cCc]
## Acceptance Criteria

- [ ] Copy does not contain any [cCc] flags
- [x] #257 
- [ ] The approved copy is added to the master copy doc by The Wildlife trust: https://docs.google.com/spreadsheets/d/1pkcokDOMpE8GIasIzsbs97xB0ZCnAi1VaQOiTC-vQ-U/edit?usp=sharing
- [ ] The copy in the app as as per the approved en and approved cy columns and marked in green
- [ ] The temporary copy column is empty

## Plan

- [x] Add the approved copy and mark green.
- [x] This will be iterative & have a few PRs against it.

## Outstanding from 20th June

- [ ] [cCc] content in the welsh versions of how to use this site and share your survey (both markdown docs)
- [ ] [cCc] content for two keys: SearchMetaDescription and IndexMetaDescription

### PR DESCRIPTION
**Pull request by katjam**
Fixes #109

## Description

- Replace copy with approved as per:

https://docs.google.com/spreadsheets/d/1pkcokDOMpE8GIasIzsbs97xB0ZCnAi1VaQOiTC-vQ-U/edit#gid=0

@geeksforsocialchange/developers

### CODE REVIEW

**Comment by katjam** [comment]
The only other cCc are in CMS copy being taken care of by #257

File: src/I18n/En.elm
Comment (r-ferrier) [inline src/I18n/En.elm:1]: line 100 - just wondering do you know is there a reason why this has three speech marks around it?

**Review by r-ferrier — APPROVED** [review]
Can't find any more [cCc] anywhere except in mentioned md files. Have left one q but it's not blocking, so approved!

File: src/I18n/En.elm
Comment (katjam) [reply src/I18n/En.elm:1]: Yes 3 quotes is a multiline string wrapper, though it looks like that was not written as one. You can use it for a single line.

**Pull request merged**